cove/views.py

```diff
@@ -102,6 +101,40 @@ def explore(request, pk):
         "created_datetime": data.created.strftime("%A, %d %B %Y %I:%M%p %Z"),
         "created_date": data.created.strftime("%A, %d %B %Y"),
     }
+    
+    schema_name = request.cove_config['schema_name']
+    replace_conversion = False
+
+    if request.current_app == 'cove-ocds':
+        schema_version = data.schema_version or request.cove_config['schema_version']
+        schema_version_choices = request.cove_config['schema_version_choices']
+        schema_version_display_choices = sorted(list({display for display, version in schema_version_choices.values()}))
```

Comment:
This makes no sense to me.  It is sorting the unique display names, but two display names could link to two different schemas.  It is the display names not the schemas that go into the database and that feels like it could go wrong.
Should we be storing 1.1 or 1.1-dev in the database? 
I also think it should be sorted by the config option order.